modules/nf-core/artic/minion/main.nf

```diff
@@ -4,17 +4,14 @@ process ARTIC_MINION {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/artic:1.2.3--pyhdfd78af_0' :
-        'biocontainers/artic:1.2.3--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/artic:1.6.1--pyhdfd78af_0' :
+        'biocontainers/artic:1.6.1--pyhdfd78af_0' }"
 
     input:
     tuple val(meta), path(fastq)
-    path  fast5_dir
-    path  sequencing_summary
+    tuple val(meta2), path(model_dir), val(model)
     path  ("primer-schemes/${scheme}/V${scheme_version}/${scheme}.reference.fasta")
     path  ("primer-schemes/${scheme}/V${scheme_version}/${scheme}.scheme.bed")
```

Comment:
What is happening with these two paths, how are they being staged here?
If they really need to be staged into this subfolder then it needs to be set with `stageAs`.

Reply:
Good question, I will update it

Reply:
Is it better ?